features/237-cms-validation/api.md

```diff
@@ -0,0 +1,146 @@
+# CMS Validation
+
+## Requirements
+
+- Prevent cms user from entering invalid data.
+- Allow developer to use common and customisable validation in cms config on field level and form level.
+- Display error message, which can be defined in config, for invalid data.
+
+## Common Use Cases
+
+- Non empty text input.
+- Text input with max length.
```

Comment:
since it is "common" use cases, shall we have a predefined validation below too?